app/src/debug/java/io/reark/rxgithubapp/utils/StethoInstrumentation.java

```diff
@@ -73,18 +70,8 @@ void initStetho() {
     @NonNull
     public OkHttpClient decorateNetwork(@NonNull final OkHttpClient httpClient) {
         Preconditions.checkNotNull(httpClient, "Http Client cannot be null.");
-
-        addInterceptor(httpClient, interceptor);
-
-        return httpClient;
+        return httpClient.newBuilder().addNetworkInterceptor(new StethoInterceptor()).build();
```

Comment:
Exactly how it was before. 
`StethoInstrumentation` receives via the constructor the interceptor it should use a.k.a. DI.
The provider of the interceptor was defined in the Dagger's module.